components/prism-ignore.js

```diff
@@ -0,0 +1,34 @@
+(function (Prism) {
+	Prism.languages.ignore = {
+		// https://git-scm.com/docs/gitignore
+		'header': {
+			pattern: /(^|[^\\])###.*/,
+			lookbehind: true,
+			alias: ['comment', 'bold']
+		},
+		'section': {
+			pattern: /(^|[^\\])##.*/,
+			lookbehind: true,
+			alias: 'comment'
```

Comment:
Maybe use `important` instead?

Reply:
I wait for your decision about sections and headers and then I'll fix it

Reply:
Removed section and header